packages/cozy-client/types/types.d.ts

```diff
@@ -19,7 +19,7 @@ export type ClientResponse = any;
 export type Manifest = any;
 export type OldCozyClient = any;
 export type NodeEnvironment = any;
-export type QueryFetchStatus = "failed" | "loading" | "pending" | "loaded";
+export type QueryFetchStatus = "loading" | "loaded" | "pending" | "failed";
```

Comment:
What do you do with this file?! You don't like it? You edit it every time! 🤣

Reply:
It's typescript that does that, it's pretty annoying.